pom.xml

```diff
@@ -21,7 +21,7 @@
 
   <groupId>io.cdap.plugin</groupId>
   <artifactId>snowflake-plugins</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>1.0.2-SNAPSHOT</version>
```

Comment:
patch version should be updated in `release/1.0` branch here - https://github.com/data-integrations/snowflake-plugins/blob/c182fc60cf77edb9af299ccc6724a9e7b24a940e/pom.xml#L24

And then the subsequent PR need to be cherry-picked in release branch for releasing the patch version.

In develop it should be a minor version bump, i.e. `1.1.0-SNAPSHOT`